shared/src/bundles/videos/types/update-video-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type CreateVideoRequestDto } from './create-video-request-dto.type.js';
+
+type UpdateVideoRequestDto = Partial<CreateVideoRequestDto>;
```

Comment:
How are we going to identify video: by its id or by userId and video name? If second option then userId and name fields are required.

Reply:
Sorry, I didn't see usage examples at first. Now I see that to delete a video we will use its id.
But in what cases we are going to change userId or url for existing video?

Reply:
For now I just implemented basic update for video and don't know about cases. We can refine it in the future, when we'll know what we will update, maybe?